### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Asanchy**
In this pull request, I added media queries for the desktop version.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -310,3 +340,135 @@ textarea {
...
+  }
+
+  hr {
+    color: red;
+    border: 1px solid blue;
+  }
+
+  footer {
+    display: block;
+    border-top: 1px solid blue;
```
Comment (keneogu) [inline style.css:472]: Your footer border-top color should be the same as the actual design. That is a gray line not blue. Please fix this.

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @Asanchy, 

Good job so far!

- All linters are passing
- You are following GitHub flow
- Good commit history
- Good PR description
- Your Design is looking good

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

- [ ] Remove the links on your readme file that is not working e.g The MIT link.
- [ ] The getting started section of the readme file should show the steps to take to clone and run the project. Please fix this.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: README.md [hunk 2]
```diff
@@ -41,7 +49,3 @@ Give a ⭐️ if you like this project!
 - Hat tip to anyone whose code was used
 - Inspiration
 - etc
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
```
Comment (gabrielyea) [inline README.md:47]: [OPTIONAL]
I'm really sorry about the misunderstanding! :pray:  You are not supposed to remove the MIT link, you should fix it so it works! Having an MIT license is part of Microverses professional repo standard and it should be in all your projects! 

- [ ] Please add the MIT licenses for your future PRs.

File: index.html [hunk 3]
```diff
@@ -42,7 +50,7 @@ <h3>My Recent Works</h3>
             </ul>
             <a href="#">See Project</a>
```
Comment (gabrielyea) [inline index.html:51]: [OPTIONAL]
- [ ] Your See Project text is not placed correctly! Try to fix it so your portfolio looks professional! :muscle: 
image

**Review by gabrielyea — APPROVED** [review]
# Approved! :partying_face: :tada: 
Hi @Asanchy , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

PD. I left some comments on things you should try to fix!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**